### PR TITLE
feat(remix-node): add `getFilePath`& `remove` methods to `NodeOnDiskFile`

### PR DESCRIPTION
**Pull request by julienmonnard**
Closes: #3021

- [ ] Docs
- [x] Tests

Testing Strategy:

One unit test for each new function, added in the remix-node package.

This is basically the same PR as #3034, but it got closed by its author before being merged. I believe this is a useful feature and would very much like to see it added.

### CODE REVIEW

File: .changeset/flat-cobras-beam.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/node": minor
+---
+
+Add a "remove" and a "getFilePath" methods to fileUploadHandler
```
Comment (machour) [inline .changeset/flat-cobras-beam.md:5]:
```suggestion
Add "remove" and "getFilePath" methods to fileUploadHandler
```

**Review by machour — APPROVED** [review]
Merci Julien for picking up this! 🙌

Just a small update to the changesetand we should be good to go.

File: .changeset/flat-cobras-beam.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/node": minor
+---
+
+Add "remove" and "getFilePath" methods to fileUploadHandler
```
Comment (MichaelDeBoey) [inline .changeset/flat-cobras-beam.md:5]: The exported class is called `NodeOnDiskFile`, so I think we should reference that name instead.

```suggestion
Add `getFilePath`& `remove` methods to `NodeOnDiskFile`
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by cstrnt** [comment]
Nice copy pasta :D

**Comment by julienmonnard** [comment]
Indeed. I'm not trying to hide that fact, it's actually mentioned in the description above.

The main difference with your PR is that this one is still open. I need that feature in my project and would like to see it merged. I was disappointed when you closed yours, and as I cannot reopen a PR, I recreated it.

**Review by mcansh — APPROVED** [review]

**Comment by julienmonnard** [comment]
Is there something more that should be done for this to be able to make it in one of the next releases?
I guess with the whole react-router/remix stuff happening it was quite tricky, but now that 1.10 wraps it up and has landed, maybe there could be a little slot for this one :)

**Review by mcansh — APPROVED** [review]

**Pull request merged**